projects/igniteui-angular/schematics/ng-add/index.spec.ts

```diff
@@ -105,13 +106,15 @@ describe('ng-add schematics', () => {
     expect(testTs).not.toContain('import \'hammerjs\';');
   });
 
-  it('should add hammer.js in angular.json build options under scripts', async () => {
+  // Hammer is optional now.
+  xit('should add hammer.js in angular.json build options under scripts', async () => {
```

Comment:
Might as well delete those test, doubt we'll ever again make it a requirement so the functionality to add it with ng-add won't be needed again